test/test_PublicInterfaceImpl.c

```diff
@@ -54,7 +54,7 @@ void setUp(void)
 
     /// Register the policy provider with some rules
     BSL_PolicyDesc_t policy_desc = { 0 };
-    policy_desc.user_data        = calloc(sizeof(BSLP_PolicyProvider_t), 1);
+    policy_desc.user_data        = BSL_CALLOC(sizeof(BSLP_PolicyProvider_t), 1);
```

Comment:
It's important that calloc() use like this uses 1 for the first argument only. Because of platform ISA, alignment may result in the wrong total size allocated. In fact if the memory is initialized immediately after alloc it can use malloc() instead.